hail/www/Makefile

```diff
@@ -4,7 +4,7 @@ GENERATED_HTML = 404.html index.html
 
 all: $(GENERATED_HTML)
 
-$(GENERATED_HTML): %.html: %.md %.xslt
+$(GENERATED_HTML): %.html: %.md %.xslt template.xslt
```

Comment:
This change isn't needed

Reply:
Both index.xslt and 404.xslt import template.xslt. Without this change, index.html and 404.html will not be rebuilt after changes to template.xslt.